src/Contracts/Models/WorkflowSpec.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace Monai.Deploy.WorkflowManager.Contracts.Models
+{
+    public class WorkflowSpec
+    {
+        [ConfigurationKeyName("name")]
```

Comment:
May want to use `JsonProperty` for models instead of `ConfigurationKeyName`

Reply:
Changed